composer.json

```diff
@@ -88,6 +89,12 @@
         "sort-packages": true,
         "platform-check": false
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "git@github.com:spryker/spryker.git"
```

Comment:
wrong repo
you must use the module in question as dependency and using split repo.